Backend/DataAccessLayer/Repositories/PointRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using System;
+using System.Data.Entity;
+using System.Linq;
+
+namespace DataAccessLayer.Repositories
+{
+    public class PointRepository
+    {
+        private Point CheckPointLimits(Point point)

```

Comment:
The naming of this function really doesn't describe what it does.

Better name would be "ValidateLatLng(Point point)"

Return value true/false

Reply:
Fixed.